apps/console/src/features/server-configurations/forms/analytics-form.tsx

```diff
@@ -0,0 +1,419 @@
+    isConnectorEnabled?: boolean;
+    /**
+     * Specifies if the form is submitting.
+     */
+    isSubmitting?: boolean;
+}
+
+const FORM_ID: string = "governance-connectors-analytics-form";
+
+/**
+ * Analytics Configuration Form.
+ *
+ * @param props - Props injected to the component.
+ * @returns Functional component.
+ */
+export const AnalyticsConfigurationForm: FunctionComponent<AnalyticsConfigurationFormPropsInterface> = (
+    props: AnalyticsConfigurationFormPropsInterface
+): ReactElement => {
+
+    const {
+        initialValues,
+        onSubmit,
+        readOnly,
+        isConnectorEnabled,
+        isSubmitting,
+        ["data-componentid"]: componentId
+    } = props;
+
+    const { t } = useTranslation();
+    const [ initialConnectorValues, setInitialConnectorValues ]
+        = useState<AnalyticsFormValuesInterface>(undefined);
+
+    /**
+     * Flattens and resolved form initial values and field metadata.
+     */
+    useEffect(() => {
+        if (isEmpty(initialValues?.properties)) {
+            return;
+        }
+
+        let resolvedInitialValues: AnalyticsFormValuesInterface = null;
+
+        initialValues.properties.map((property: ConnectorPropertyInterface) => {
+            switch (property.name) {
+                case ServerConfigurationsConstants.ANALYTICS_HOST:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        receiver: property.value
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_BASIC_AUTH_ENABLE:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        basicAuthEnabled: CommonUtils.parseBoolean(property.value)
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_BASIC_AUTH_USERNAME:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        basicAuthUsername: property.value
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_BASIC_AUTH_PASSWORD:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        basicAuthPassword: property.value
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_HTTP_CONNECTION_TIMEOUT:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        httpConnectionTimeout: parseInt(property.value)
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_HTTP_READ_TIMEOUT:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        httpReadTimeout: parseInt(property.value)
```

Comment:
If property.value is null | undefined, this will output NaN

Reply:
Fixed with 1d3ef8fd4f615de27060c959ea91a56ca1f3158f